src/client/clientlauncher.cpp

```diff
@@ -36,6 +40,11 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "clientlauncher.h"
 #include "version.h"
 
+// irr::ELL_DEBUG was added in irrlicht 1.8
+#if (IRRLICHT_VERSION_MAJOR == 1 && IRRLICHT_VERSION_MINOR < 8)
+#define ELL_DEBUG ELL_INFORMATION
```

Comment:
this is a horrible way of doing this, move the `#ifdef` down into the actual code